dev-libs/spdlog/spdlog-1.9.2-r1.ebuild

```
@@ -31,7 +31,7 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
-PATCHES=( "${FILESDIR}/${PN}-force_external_fmt.patch" )
+PATCHES=( "${FILESDIR}/${PN}-force_external_fmt.patch" "${FILESDIR}/${P}-fix-clone-test.patch" )
```

Please break into multi lines - makes it easier to drop or add patches
```suggestion
PATCHES=(
    "${FILESDIR}/${PN}-force_external_fmt.patch"
    "${FILESDIR}/${P}-fix-clone-test.patch"
)
```